### PR TITLE
Prefetch While Computing

### PR DESCRIPTION
**Pull request by KarlLeell**

### CODE REVIEW

File: src/storage/gpu_swap_prefetch.cc
Comment (Sotskin) [inline src/storage/gpu_swap_prefetch.cc:132]: Add algorithm discription

File: src/storage/gpu_swap_prefetch.cc
Comment (Sotskin) [inline src/storage/gpu_swap_prefetch.cc:99]: Add algorithm description

**Review by Sotskin — CHANGES_REQUESTED** [review]
Looks good. Please add some simple descriptions for the algorithm

**Review by fegin — APPROVED** [review]
You may have a conflict (benchmark/run.sh) when merging this pull request because of @Sotskin 's pull request. Make sure you merge this pull request after he merges and resolves the conflict if there is one.

**Pull request merged**
